Buildings/Resources/Data/Templates/Validation/systems.json

```diff
@@ -483,5 +483,173 @@
                 }
             }
         }
+    },
+    "CHW_1": {
+        "ChilledWater": {
```

Comment:
The function calls such as `dat.getReal(varName=id + ".ChillerGroup.dpCW_nominal.value")` will be used by the tool to generate the equipment schedule as specified at https://docs.google.com/document/d/1wPQ4qIbOET5llsK3XfPuYusFWBzP3s7KMzQfhMb8mVs/edit?usp=sharing.
Having camel case instead of human readable strings means that we will need to develop a camel case parser, with the issue of capital letters that sometimes need to be kept as capital, sometimes not.
What is the programmatic advantage of camel case here?

Reply:
OK to add a `description` field for each object and to use names with no spaces in them.
Also note that I refactored the schema to add the type of system as the top-level object, for instance:
```
{
    "Air handlers and fans": {
        "VAV_1": {...
```
In Modelica code, I now have two declarations within each interface class.
```
  inner parameter String tag
    "System tag"
    annotation (
      Evaluate=true,
      Dialog(group="Configuration"));
  final inner parameter String id = "Air handlers and fans." + tag
    "System tag with system type"
    annotation (
      Evaluate=true,
      Dialog(group="Configuration"));
```
If that sounds good to you, please refactor the CHW plant template accordingly.

Reply:
I agree with you and I will remove the additional layer. The system type may just be added as an attribute as you mention.